etc/ci/upload_nightly.sh

```diff
@@ -25,9 +25,8 @@ upload() {
     s3cmd cp "${package_upload_path}" "${nightly_upload_dir}/servo-latest.${3}"
 }
 
-
```

Comment:
This is resolved in the next commit (haven't done any squashing yet in order to separate out my fixes and improvements).